go.mod

```diff
@@ -1,18 +1,20 @@
 module github.com/rwx-research/mint-cli
```

Comment:
The changes in here were a result of trying to install gomega/ginkgo and running `go mod tidy`. I don't think it's impactful.